Assets/Scripts/CurrentPickup.cs

```diff
@@ -19,6 +19,13 @@ void OnTriggerEnter2D(Collider2D collision)
                 Player.player.coins += quantity;
                 Debug.Log(Player.player.coins);
             }
+
+            if (currentObject == PickupObject.HP)
+            {
+                Player.player.health += quantity;
```

Comment:
I would consider adding a AddHealth function to the player instead of just publicly exposing and editting their health. Can their health go over 100% now?